src/strings/PowerFxResources.en-US.resx

```diff
@@ -4694,4 +4694,24 @@
     <value>The specified data source cannot be used with this function.</value>
     <comment>Error Message.</comment>
   </data>
-</root>
\ No newline at end of file
+  <data name="ErrDivByZero" xml:space="preserve">
+    <value>Invalid operation: division by zero.</value>
```

Comment:
>Invalid [](http://example.com/codeflow?start=11&length=7)

The key challenge here will be sharing these with the JS resources in PowerApp's JScript runtime.

Reply:
This has been discussed internally.